<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>551</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="Canva" name="widget" native="true"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>221</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_box">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Box</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>x</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider_BX">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumber_5"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider_BY">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumber_6"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>z</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider_BZ">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumber_7"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_Elipsoide">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Elipsoid</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>R x</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider_RX">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumber_4"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>R y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider_RY">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumber_3"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>R z</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="horizontalSlider_RZ">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="lcdNumber_2"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>380</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Sphere</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Radius</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderRadiusSphere">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>781</width>
      <height>220</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Coordinates</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_20">
         <item>
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Axis Z</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider_z">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumber_13"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_22">
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Axis Y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumber_y"/>
         </item>
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Axis X</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumber_x"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_RGB">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_16">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Color</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_19">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>R</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="horizontalSlider_R">
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="lcdNumber_8"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_18">
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>G</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="horizontalSlider_G">
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="lcdNumber_9"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_17">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>B</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="horizontalSlider_B">
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="lcdNumber_10"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_color_preview">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNovo_Projeto"/>
    <addaction name="actionSalvar"/>
    <addaction name="actionSair"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionNovo_Projeto"/>
   <addaction name="actionSalvar"/>
   <addaction name="actionApagar_Desenho"/>
   <addaction name="actionputVoxel"/>
   <addaction name="actionputSphere"/>
   <addaction name="actionputBox"/>
   <addaction name="actionputEllipsoid"/>
   <addaction name="actioncutVoxel"/>
   <addaction name="actioncutSphere"/>
   <addaction name="actioncutBox"/>
   <addaction name="actioncutEllipsoid"/>
  </widget>
  <action name="actionNovo_Projeto">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/add.png</normaloff>:/icones/add.png</iconset>
   </property>
   <property name="text">
    <string>Novo Projeto</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/sair.png</normaloff>:/icones/sair.png</iconset>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionputVoxel">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/ponto.png</normaloff>:/icones/ponto.png</iconset>
   </property>
   <property name="text">
    <string>putVoxel</string>
   </property>
  </action>
  <action name="actionputSphere">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/esfera.png</normaloff>:/icones/esfera.png</iconset>
   </property>
   <property name="text">
    <string>putSphere</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/baixar.png</normaloff>:/icones/baixar.png</iconset>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
  <action name="actionputBox">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/caixa.png</normaloff>:/icones/caixa.png</iconset>
   </property>
   <property name="text">
    <string>putBox</string>
   </property>
  </action>
  <action name="actionApagar_Desenho">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/apagar.png</normaloff>:/icones/apagar.png</iconset>
   </property>
   <property name="text">
    <string>Apagar Desenho</string>
   </property>
  </action>
  <action name="actioncutVoxel">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/xponto.png</normaloff>:/icones/xponto.png</iconset>
   </property>
   <property name="text">
    <string>cutVoxel</string>
   </property>
  </action>
  <action name="actioncutSphere">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/Xesfera.png</normaloff>:/icones/Xesfera.png</iconset>
   </property>
   <property name="text">
    <string>cutSphere</string>
   </property>
  </action>
  <action name="actioncutBox">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/xcaixa.png</normaloff>:/icones/xcaixa.png</iconset>
   </property>
   <property name="text">
    <string>cutBox</string>
   </property>
  </action>
  <action name="actionputEllipsoid">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/elipse.png</normaloff>:/icones/elipse.png</iconset>
   </property>
   <property name="text">
    <string>putEllipsoid</string>
   </property>
  </action>
  <action name="actioncutEllipsoid">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/Xelipse.png</normaloff>:/icones/Xelipse.png</iconset>
   </property>
   <property name="text">
    <string>cutEllipsoid</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Canva</class>
   <extends>QWidget</extends>
   <header>canva.h</header>
   <container>1</container>
   <slots>
    <signal>enviaPosicao(int,int)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>horizontalSlider_BX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_5</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>708</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>738</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_BY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_6</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>716</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>734</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_BZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_7</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>659</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>728</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_RX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_4</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>714</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>744</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_RY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_3</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>710</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>741</x>
     <y>384</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_RZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>670</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>752</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderRadiusSphere</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>714</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>744</x>
     <y>514</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_R</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_8</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>707</x>
     <y>596</y>
    </hint>
    <hint type="destinationlabel">
     <x>736</x>
     <y>596</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_G</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_9</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>701</x>
     <y>632</y>
    </hint>
    <hint type="destinationlabel">
     <x>748</x>
     <y>635</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_B</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_10</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>703</x>
     <y>669</y>
    </hint>
    <hint type="destinationlabel">
     <x>741</x>
     <y>669</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_z</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_13</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>693</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>698</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSair</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>quit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionputVoxel</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>putVoxel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionputSphere</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>putSphere()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widget</sender>
   <signal>enviaPosicao(int,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>draw3d(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widget</sender>
   <signal>enviaPosicao(int,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateAxis(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_z</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>set_Axis_z(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>662</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_R</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>setRed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>568</x>
     <y>562</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_G</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>setGreen(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>568</x>
     <y>599</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_B</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>setBlue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>568</x>
     <y>635</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSalvar</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionputBox</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>putBox()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionApagar_Desenho</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>limpar_voxel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionputEllipsoid</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>putEllipsoid()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actioncutVoxel</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>cutVoxel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actioncutSphere</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>cutSphere()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actioncutBox</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>cutBox()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actioncutEllipsoid</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>cutEllipsoid()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>quit()</slot>
  <slot>putVoxel()</slot>
  <slot>putSphere()</slot>
  <slot>setRed(int)</slot>
  <slot>setGreen(int)</slot>
  <slot>setBlue(int)</slot>
  <slot>draw3d(int,int)</slot>
  <slot>save()</slot>
  <slot>updateAxis(int,int)</slot>
  <slot>set_Axis_z(int)</slot>
  <slot>putBox()</slot>
  <slot>limpar_voxel()</slot>
  <slot>putEllipsoid()</slot>
  <slot>cutVoxel()</slot>
  <slot>cutSphere()</slot>
  <slot>cutBox()</slot>
  <slot>cutEllipsoid()</slot>
 </slots>
</ui>
